components/MobileNavigation/MobileNavigation.styled.js

```diff
@@ -0,0 +1,45 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export const StyledNavigation = styled.nav`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: var(--mobile-navigation-height);
+  background-color: black;
+  border: 2px solid black;
+`;
+
+export const UnorderedList = styled.ul`
+  display: flex;
+  gap: 2px;
+  height: 100%;
+`;
+
+export const ListItem = styled.li`
+  list-style: none;
+  width: 100%;
+  text-align: center;
```

Comment:
You don't really need to use the `text-align` property here because you position your icon (Link) in your `StyledLink` styled component.
```suggestion

```

Reply:
Thx. Deleted it